src/components/Button/test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { renderWithClient } from 'testUtils';
+import { Button, ButtonUI } from '.';
+
+describe('ButtonUI', () => {
+    test('renders UI', () => {
+        render(<ButtonUI>ButtonUI</ButtonUI>);
+        screen.getByText('ButtonUI');
+    });
+});
+
+describe('Button', () => {
```

Comment:
I think for this one it'd at least be good to fire a click event and make sure the onClick gets called. Can be nice to test that the `kind` generates the expected classes too.